README.md

```
@@ -62,30 +62,49 @@ The Relink container provides a complete crosslinking mass spectrometry analysis
 
 | Container Type | Tag    | URL                                      |
 | -------------- | ------ | ---------------------------------------- |
-| Docker         | 1.0.0  | `ghcr.io/bigbio/relink:1.0.0`            |
+| Docker         | 1.1.0  | `ghcr.io/bigbio/relink:1.1.0`            |
 | Docker         | latest | `ghcr.io/bigbio/relink:latest`           |
-| Singularity    | 1.0.0  | `oras://ghcr.io/bigbio/relink-sif:1.0.0` |
+| Singularity    | 1.1.0  | `oras://ghcr.io/bigbio/relink-sif:1.1.0` |
+
+All bundled tools are exposed through a single `relink` dispatcher on `PATH`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)